mailserver/mailserver.go

```diff
@@ -334,6 +369,8 @@ func (s *WMailServer) SyncMail(peer *whisper.Peer, request whisper.SyncMailReque
 
 	defer recoverLevelDBPanics("SyncMail")
 
+	requestID := fmt.Sprintf("%d", time.Now().UnixNano())
```

Comment:
to be absolutely paranoid, maybe we can make it like
```suggestion
	requestID := fmt.Sprintf("%d-%d", time.Now().UnixNano(), rand.Intn(1000))
```

Reply:
Why not :) I copied it from the previous implementation.